### PR TITLE
Fixes #309 Show relevant error messages

### LINKED ISSUE #309: Show relevant error messages in chat box
This is a follow up to issue: https://github.com/fossasia/susi_android/issues/306

Currently there is a general error message if something went wrong in Susi "an error ocurred. please try again."

As a matter of fact the server seems to work perfectly fine and it is unclear what the error is. We need a simple way to understand the cause without having to browse through logs at this stage of development.

Please show real error message/reason for errors instead of "there was an error".

@mejariamol @amitiwary999 Please help if possible.

### PR DESCRIPTION
**Pull request by mejariamol**
Fixes issue #309 

Changes: Two errors were identified and shown accordingly:

1. Invalid Access Token

2. Internet Connectivity Problem

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -634,7 +634,7 @@ public void execute(Realm bgRealm) {
                                         });
                                         rvChatFeed.getRecycledViewPool().clear();
                                         recyclerAdapter.notifyItemChanged((int) id);
-                                        addNewMessage(getString(R.string.error_occurred_try_again), false, false, false, null);
+                                        addNewMessage("Invalid Access Token !", false, false, false, null);
```
Comment (jig08) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: @mejariamol Kindly use `strings.xml` for this purpose.

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 2]
```diff
@@ -671,7 +671,7 @@ public void execute(Realm bgRealm) {
                                     });
                                     rvChatFeed.getRecycledViewPool().clear();
                                     recyclerAdapter.notifyItemChanged((int) id);
-                                    addNewMessage(getString(R.string.error_occurred_try_again), false, false, false, null);
+                                    addNewMessage("Internet Connectivity Problem !", false, false, false, null);
```
Comment (jig08) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Here too.

**Comment by mejariamol** [comment]
@jig08 used `string.xml`.

**Comment by jig08** [comment]
@mejariamol Squash the commits too.

**Comment by mayank408** [comment]
@jig08 @mejariamol  I dont think we should give a message like Invalid acess token .. Its our concern not the one that is shown to the user . It will be a bad user experience.

**Comment by mejariamol** [comment]
@mayank408 It is not going to be a permanent thing. At this stage, it would be better if errors shown are more specific until they are fixed.

**Comment by mejariamol** [comment]
@jig08 squashed.

**Comment by daminisatya** [comment]
@mejariamol @jig08 Is there a possibility to make these messages more user friendly? 

```
Invalid Access Token
Internet Connectivity Problem
```

@mejariamol Can you explain about the ```Invalid access token``` message?

**Comment by mejariamol** [comment]
@daminisatya When the expired access token is used in the request, server sends null body in the SusiResponse. At this stage there is no any implementation for renewing expired access token. Issue #311

**Comment by daminisatya** [comment]
👍 cool merging it.

**Pull request merged**